proto/README.md

```diff
@@ -52,10 +55,14 @@ confidnece | - | float | 0 to 1 probability of the object classification.
 bbox | - | [BoundingBox](#boundingbox) | Bounding box of the object.
 velocity | m/s | Vector3 | XYZ velocity of the object.
 tracking_status | - | [TrackingStatus](https://github.com/seoulrobotics/sensr_proto/blob/master/type.proto) | tracking status.
+yaw_rate | - | float | Object rotation angle along the Z axis. (radian)
+time_offset | - | [Duration](https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/duration) | Life time of the object.
+retro_reflective | - | bool | A flag to indicate the object has retro reflective point or not.
```

Comment:
This is not really correct. The flag shows if it is classified as retro reflective. It can have several retro reflective points but is not classified as retro reflective.

Reply:
"A flag to show if the object is classified as retro reflective."
Is this OK?

Reply:
Yes, that works :+1: